eyeshade/controllers/accounts.js

```diff
@@ -121,18 +123,19 @@ v1.getBalances =
 }
 
 /*
-   GET /v1/accounts/earnings/{type}/contributions
+   GET /v1/accounts/earnings/{type}/total
 */
 
 v1.getEarningsTotals =
 { handler: (runtime) => {
   return async (request, reply) => {
+    const debug = braveHapi.debug(module, request)
     let { type } = request.params
     let {
       order,
       limit
     } = request.query
-
+    debug('query', request.query)
```

Comment:
is this duped by our query method which also logs?

Reply:
ah, yes, can fix